card/lib/cardio.rb

```diff
@@ -50,26 +48,19 @@ def set_config config
   
     def set_paths paths
       @@paths = paths
-      paths.add 'tmp/lib'
-      paths.add 'tmp/set'
-      paths.add 'tmp/set_pattern'
-      paths.add 'db/migrate_deck_cards', :with=>'db/migrate_cards'
 
-      add_gem_path 'mod',      :with => 'mod'
-      add_gem_path "db"
+      add_gem_path 'tmp/set',               :root => root, :with => '/tmp/set'
+      add_gem_path 'tmp/set_pattern',       :root => root, :with => '/tmp/set_pattern'
```

Comment:
??  why the heck would you call add_gem_path to add a deck path?  please restore